src/dfi/consensus/xvm.h

```diff
@@ -11,8 +11,7 @@ struct CEvmTxMessage;
 struct CTransferDomainMessage;
 
 enum class VMDomain : uint8_t {
-    NONE = 0x00,
-    // UTXO Reserved
+    AUTO = 0x00,
```

Comment:
I'd still keep this as `None` to represent the Unknown state on the enum. 

The auto is just purely on the RPC side, where it shifts unknown state into a deterministic known state. Wouldn't add an auto state to the protocol message itself.